src/Models/Menu.php

```diff
@@ -90,4 +99,39 @@ public function buildTree($elements, $parentId = 0)
 
         return $branch;
     }
+
+    /**
+     * @return array
+     */
+    public static function getRoutes($type = 'admin')
+    {
+        $routes = ['' => '- Select -'];
+
+        foreach (\Route::getRoutes() as $route) {
+            if (!$route->getName() || !in_array('GET', $route->methods())) {
+                continue;
+            }
+
+            $action = array_last(explode('.', $route->getName()));
+            $name = str_replace(['.', 'admin', 'index'], ' ', $route->getName());
+            if ($type === 'admin' &&
+                in_array('laradium', $route->middleware()) &&
+                in_array($action, ['index', 'create', 'dashboard']) &&
+                $route->getName() !== 'admin.index'
+            ) {
+                $routes[$route->getName()] = ucfirst(trim($name));
+            } else if ($type === 'public' &&
```

Comment:
Conditions can be moved to separate method and `else if` can be removed, you just need to add an additional condition to the first `if`: `OR`